bindings/rust/build.rs

```diff
@@ -30,38 +21,85 @@ fn main() {
 
     eprintln!("Using FIELD_ELEMENTS_PER_BLOB={}", field_elements_per_blob);
 
-    // Deleting any existing assembly and object files to ensure that compiling with a different
-    // feature flag changes the final linked library file.
-    let obj_file = root_dir.join("src").join("c_kzg_4844.o");
-    if obj_file.exists() {
-        std::fs::remove_file(obj_file).unwrap();
+    let c_src_dir = root_dir.join("src");
+
+    let mut cc = cc::Build::new();
+
+    let file_vec = vec![c_src_dir.join("c_kzg_4844.c")];
+
+    /*
+     * BLST env
+     */
+
+    let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+
+    if target_arch.eq("x86_64") || target_arch.eq("aarch64") {
+    } else {
+        cc.define("__BLST_NO_ASM__", None);
     }
```

Comment:
This is a neat section. I think you should add a `force-arm-feature-crypto` (maybe with a better name) for ARM systems. Some ARM systems (such as new the M1/M2 Macs) have specialized crypto instructions and blst supports that. For reference, [this is where this is checked for in blst](https://github.com/supranational/blst/blob/a7fd1f584d26b0ae6cdc427976ea1d8980f7e15d/src/sha256.h#L14-L15).

Reply:
I actually removed all of this until it's clear how c-kzg maps to blst feature flags. This whole code is executed by the blst build script and is then unnecessary